src/components/SyntaxHighlighter/yql.ts

```diff
@@ -1,82 +0,0 @@
-import {
-    builtinFunctions,
-    keywords,
-    typeKeywords,
-} from 'monaco-yql-languages/build/yql/yql.keywords';
```

Comment:
These constants are not valid yet, and it won't be undated in future. Soon it will be removed from the library.